upload_stage_rpms

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -e
```

Comment:
This script is designed to work with this change https://github.com/theforeman/foreman-infra/pull/1948 that allows infra users to rsync as `yumrepostage` user.